resources/config/conf.yaml

```diff
@@ -0,0 +1,12 @@
+plugins:
+  trivy:
+    exit-code: 0
+    timeout: 5m0s
+    severity: HIGH,CRITICAL
+    ignore-unfixed: true
+    security-checks: vuln,config
```

Comment:
Maybe we should make this a yaml list?

Reply:
I'm here referring trivy buildkite plugin README.md